code/__HELPERS/game.dm

```diff
@@ -395,6 +395,48 @@
 
 	return new /datum/projectile_data(src_x, src_y, time, distance, power_x, power_y, dest_x, dest_y)
 
+/proc/pollCultists()
+	var/time_passed = world.time
+	set waitfor = FALSE
+	var/list/yes_voters = new
+	var/list/cult_total = new
+	for(var/mob/M in mob_list)
+		spawn(0)
+		if(iscultist(M))
+			M << 'sound/misc/notice2.ogg' //Alerting them to their consideration
+			M.verbs -= /mob/living/proc/cult_master
+			switch(askuser(M,"[usr] seeks to lead your cult, do you support  them?","Please answer in 20 seconds!","Yes","No","Abstain", StealFocus=0, Timeout=200))
+				if(1)
+					M << "<span class='notice'>Choice registered: Yes.</span>"
+					if((world.time-time_passed)>200)
+						M << "<span class='danger'>Sorry, you were too late for the consideration!</span>"
+						M << 'sound/machines/buzz-sigh.ogg'
+					else
+						yes_voters += M
+						cult_total += M
+				if(2)
+					M << "<span class='danger'>Choice registered: No.</span>"
+					cult_total += M
+				if(3)
+					M << "<span class='danger'>Choice registered: Abstain.</span>"
+	sleep(200)
+	if(yes_voters.len > (cult_total.len/2))
+		var/datum/action/innate/cultmast/FinalReckoning= new()
+		var/datum/action/innate/cultmark/Mark= new()
+		FinalReckoning.Grant(usr)
+		Mark.Grant(usr)
+		usr.mind.special_role = "Cult Master"

```

Comment:
any code using usr, besides topics, tgui and certain admin stuff, is complete trash.
usr is whoever called a verb, the fact it's available in procs has always been unwanted, even Lummox will tell you to avoid it, since when you call a verb, there's always a usr, when you call a proc... there's no guarantee even remotely that the usr is correct or if it even exists at all.